cli/testdata/coder_templates_versions_promote_--help.golden

```diff
@@ -0,0 +1,22 @@
+coder v0.0.0-devel
+
+USAGE:
+  coder templates versions promote [flags]
```

Comment:
I know the output `[flags]` technically includes my usage comment earlier. Given they are required, I wonder if we should include them more explicitly.